crates/oidc-client/src/requests/authorization_code.rs

```diff
@@ -107,17 +202,22 @@ struct FullAuthorizationRequest {
 
 /// Build the authorization request.
 fn build_authorization_request(
-    authorization_data: AuthorizationRequestData<'_>,
+    authorization_data: AuthorizationRequestData,
+    code_challenge_methods_supported: Option<&[PkceCodeChallengeMethod]>,
```

Comment:
Honestly we should just clone it, it's small enough to not matter.
I'd rather have an ergonomic API than a weird one just to avoid a few avoidable clones.